assets/js/tienichsv.js

```
@@ -1,194 +1,447 @@
+        const totalCredit = d.data.reduce((acc, curr) => {
+          if (curr.isHead) return acc;
+          if (curr.isIgnore) return acc;
+          return acc + parseInt(curr.credit);
+        }, 0);
+
+        const avg = d.data.reduce(
+          (acc, curr) => {
+            const point = parseFloat(curr.point);
+            const credit = parseInt(curr.credit);
+
+            if (curr.isIgnore || isNaN(point) || isNaN(credit)) return acc;
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)